inlineplz/decorators.py

```diff
@@ -22,5 +21,34 @@ def _create_linter(klass, config):
     return klass
 
 
-def linter(config):
-    return partial(_create_linter, config=config)
+def linter(
+    name,
+    language,
+    install,
+    help,
```

Comment:
[inline-plz]: `prospector: pylint: Redefining built-in 'help' (redefined-builtin)`